src/main/resources/init_db.sql

```diff
@@ -7,3 +7,13 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+create table if not exists drivers
+(
+    id             bigint auto_increment
+    primary key,
+    name           varchar(255)         not null,
+    license_number varchar(255)         not null,
+    is_deleted     tinyint(1) default 0 not null
+    );
+
```

Comment:
```suggestion
create table if not exists `drivers`
(
    `id`             bigint auto_increment primary key,
    `name`           varchar(255)         not null,
    `license_number` varchar(255)         not null,
    `is_deleted`     tinyint default '0' not null
    );

```
Better use the same style as the previous query